src/onnx/parse_prefix_scan.cpp

```diff
@@ -0,0 +1,70 @@
+#include <migraphx/onnx/op_parser.hpp>
+#include <migraphx/ranges.hpp>
+#include <migraphx/instruction.hpp>
+#include <migraphx/make_op.hpp>
+
+namespace migraphx {
+inline namespace MIGRAPHX_INLINE_NS {
+namespace onnx {
+
+instruction_ref parse_prefix_scan_oper(const std::string& op_name,
+                                  const onnx_parser& parser,
+                                  onnx_parser::node_info info,
+                                  std::vector<instruction_ref> args)
+{
+    int64_t axis = 0; 
+    migraphx::argument in = args[1]->eval();
+    if (in.empty())
+    {
+        std::cout << "No axis given. Using axis=0" << std::endl;
```

Comment:
Shouldn't this be an error if the axis is missing?

Reply:
The only way I've gotten the onnx CumSum node in gen_onnx.py to actually work is to pass in 'axis' as a tensor_value_info rather than an actual tensor or constant. So when I run onnx_test.cpp, having it throw an error causes the test to fail. If I can figure out how to get an actual value for 'axis' into the onnx CumSum node, then I'll fix this.

Reply:
@kahmed10 @scxiao Are you able to help Alan generate the correct onnx?

Reply:
I refined the script and committed the changes. Not sure whether that is what you want.